Bugzilla/Search.pm

```diff
@@ -1435,8 +1435,9 @@ sub _translate_join {
   if ($extra_condition) {
     $extra_condition = " AND $extra_condition";
   }
-
-  my @join_sql = "$join JOIN $table AS $name"
+  my $sql_table
+    = $table eq 'groups' ? Bugzilla->dbh->quote_identifier($table) : $table;
```

Comment:
We could simplify this by unconditionally quote-ing it, though I'm happy either way.

Reply:
I tried this in some cases the $table value is actually another string of SQL such as "foo JOIN ON (SELECT blah FROM blar)" So I can only quote if table is groups.